pyproject.toml

```diff
@@ -0,0 +1,86 @@
+[project]
+name = "adept"
+description = "Automatic-Differentiation-Enabled Plasma Transport in JAX"
+readme = "README.md"
+authors = [
+    {name = "Archis Joglekar/Ergodic LLC", email="archis@ergodic.io"},
+]
+license = "MIT"
+license-files = ["LICENSE*"]
+requires-python = ">=3.10"
+dependencies = [
+        "diffrax",
+        "matplotlib",
+        "scipy",
+        "numpy",
+        "tqdm",
+        "xarray",
+        "mlflow",
+        "flatdict",
+        "h5netcdf",
+        "optax",
+        "boto3",
+        "pint",
+        "mlflow_export_import",
```

Comment:
this is no longer needed and removed from the requirements in the most recent PR